packages/manager/modules/nasha/src/dashboard/partition/partition.controller.js

```diff
@@ -0,0 +1,3 @@
+import NashaDashboardController from '../dashboard.controller';
+
+export default class NashaDashboardPartitionController extends NashaDashboardController {}
```

Comment:
Useless controller here

Reply:
Do you mean useless extends ?
Because the controller is not ^^